lightning/src/ln/channelmanager.rs

```diff
@@ -1668,6 +1698,31 @@ where
 		self.list_channels_with_filter(|&(_, ref channel)| channel.is_live())
 	}
 
+	/// Gets a list of payments in random order that are currently pending resolution.
```

Comment:
Similar here regarding pending resolution. Can take from `RecentPaymentDetails` docs and also:

s/Gets a list of/Returns recent
s/random/an undefined